src/initialize_dae.jl

```diff
@@ -250,12 +250,14 @@ function _initialize_dae!(integrator, prob::DAEProblem,
     f((u-u0)/dt,u,p,t)
   end
 
-  nlequation! = @closure (out,u) -> out .= nlequation_oop(u)
+  nlequation = @closure (u,_) -> nlequation_oop(u)
 
   resid = f(integrator.du,u0,p,t)
   integrator.opts.internalnorm(resid,t) <= integrator.opts.abstol && return
 
-  integrator.u = nlsolve(nlequation!, u0).zero
+  nlprob = NonlinearProblem(nlequation,u0)
+  sol = solve(nlprob,NewtonRaphson(autodiff=false))
```

Comment:
Can we get autodiff from `unwrap_alg(integrate)` or sth?

Reply:
No, because NonlinearSolve will need an update for its autodiff to be compatible on the same functions do to how it under-types the duals. It needs standard tags, etc. That's a big update but I'll do it next.